src/models/Roadwork.ts

```diff
@@ -0,0 +1,26 @@
+import { Document, model, Schema } from 'mongoose';
+
+interface IRoadwork extends Document {
```

Comment:
`I...` for interface?

Reply:
`I` for interface yes. this is common for interfaces in react-native, so I adopted it here, as it also gets rid of the naming conflict with the model itself.

Reply:
never heard that "type in variable name" is called the [Hungarian notation](https://en.wikipedia.org/wiki/Hungarian_notation).